clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -39,6 +39,7 @@ import (
 
 const (
 	defaultPodStartupLatencyThreshold = 5 * time.Second
+	defaultSchedulerName              = "corev1.DefaultSchedulerName"
```

Comment:
It should be `corev1.DefaultSchedulerName` and not `"corev1.DefaultSchedulerName"`, right?

Reply:
I think we are passing it as a string, if we use `corev1.DefaultSchedulerName` then a bunch of tests fails.

Reply:
Before your change, "source" parameter was hardcoded to `corev1.DefaultSchedulerName` (not `"corev1.DefaultSchedulerName"`) and it was working.

Reply:
Thanks for the feedback, very valuable. I have made the changes and all tests pass.